deploy/examples/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: rook-ceph
+namespace: rook-ceph # namespace:cluster
```

Comment:
I don't think this is needed in the examples folder, according to #11281 it was just added for csv generation. I'd suggest we move this file into the `build/csv` folder next to the `csv-gen.sh` script, and not worry about adding the `# namespace:cluster` comment. @subhamkrai ok?

Reply:
@travisn I tried to move the file but then found that the k8s `kustomize` requieres the resources (crds.yaml, etc) to be at the same or below as the directory where kustomization.yaml is located. So I'm just reverting the changes on the file and keeping it on the same location.